corehq/apps/app_manager/suite_xml/sections/details.py

```diff
@@ -130,7 +130,8 @@ def get_section_elements(self):
                         )
                         if d:
                             elements.append(d)
-                            if self.app.supports_case_list_optimizations:
+                            if (self.app.supports_case_list_optimizations
+                                    and module.show_case_list_optimization_options):
```

Comment:
open to considering renaming `show_case_list_optimization_options` if that seems worth it to
`support_case_list_optimization_options` or `add_case_list_optimization_options`.

Reply:
Actually nevermind. May be I am overthinking this.